.github/workflows/docat.yml

```
@@ -3,7 +3,7 @@ name: docat ci
 on: [push]
```

```suggestion
on: [push, pull_request]
```

I think we require a `pull_request` trigger, too.

---

sure i checked the docs and idk lets add it